AIDevGallery/MainWindow.xaml.cs

```diff
@@ -82,7 +82,7 @@ public void Navigate(string Tag, object? obj = null)
                 Navigate(typeof(APISelectionPage), obj);
                 break;
             case "contribute":
-                _ = Launcher.LaunchUriAsync(new Uri("https://aka.ms/ai-dev-gallery"));
+                _ = Launcher.LaunchUriAsync(new Uri("https://github.com/microsoft/ai-dev-gallery"));
```

Comment:
```suggestion
                _ = Launcher.LaunchUriAsync(new Uri("https://aka.ms/ai-dev-gallery-repo"));
```

We use these aka.ms links to help us track usage.